apps/docs/pages/guides/auth/auth-mfa.mdx

```diff
@@ -625,7 +705,7 @@ Currently recognized methods are:
 - `otp` - any one-time password based sign in (email code, SMS code, magic
   link).
 - `oauth` - any OAuth based sign in (social login).
-- `mfa/totp` - a TOTP additional factor.
```

Comment:
I think this is `mfa/totp` no?

Reply:
oh I think it's just `totp` 

https://github.com/supabase/gotrue/blob/master/models/factor.go#L39